query/tests/it/servers/http/http_query_handlers.rs

```diff
@@ -543,13 +546,16 @@ async fn post_json_to_endpoint(
     let uri = "/v1/query";
     let content_type = "application/json";
     let body = serde_json::to_vec(&json)?;
+    let basic = headers::Authorization::basic("root", "");
 
     let response = ep
         .call(
             Request::builder()
                 .uri(uri.parse().unwrap())
                 .method(Method::POST)
                 .header(header::CONTENT_TYPE, content_type)
+                .header(header::HOST, "localhost")
```

Comment:
not safe  if client can set it

Reply:
I understand that the client cannot set it, but I removed it.